backend/hitas/templates/unconfirmed_maximum_price.jinja

```diff
@@ -54,21 +54,35 @@
             <td>Huoneiston ensimmäinen kaupantekoajankohta</td>
             <td>{{ apartment.prices.first_purchase_date|format_date }}</td>
         </tr>
+        <tr>
+            <td>Rakennuttajalta tilatut lisä- ja muutostyöt</td>
+            <td>{{ apartment.prices.construction.additional_work|intcomma }}</td>
+        </tr>
+        {% set cpi_bigger = prices.construction_price_index.value > prices.market_price_index.value %}
+        {% if old_hitas_ruleset %}
+        <tr>
+            <td>Rakennusaikaisen koron rahasumma</td>
+            {% if prices.construction_price_index.maximum or (prices.surface_area_price_ceiling.maximum and cpi_bigger) %}
```

Comment:
Wouldn't just `{% if cpi_bigger %}` do the exact same thing 😄

Reply:
Yes, I'll change this.